cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -177,6 +177,20 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) {
 	podsToReset, unschedulablePodsToHelp := SlicePodsByPodScheduledTime(allUnschedulablePods, allNodesAvailableTime)
 	ResetPodScheduledCondition(a.AutoscalingContext.ClientSet, podsToReset)
 
+	if autoscalingContext.MinExtraCapacityRate > 0.0 {
+		scheduledPlaceholderPods, unscheduledPlaceholderPods, err := CreateAndSchedulePlaceholderPods(readyNodes, allScheduled, autoscalingContext, a.PredicateChecker)
+
+		if err != nil {
+			glog.Errorf("Failed to schedule placeholder pods: %v", err)
+			return
```

Comment:
@mumoshu this return fails in compilation. This `errors.ToAutoscalerError(errors.ApiCallError, err)` works